gem/lib/pupperware/spec_helper.rb

```diff
@@ -350,6 +350,20 @@ def emit_logs
     get_containers.each { |id| emit_log(id) }
   end
 
+  def kill_service_and_wait_for_return(service: , process:)
```

Comment:
Should probably be explicit here and default these to `nil` like we do elsewhere.